be1-go/hub/election.go

```diff
@@ -87,13 +90,14 @@ func (c *laoChannel) createElection(msg message.Message) error {
 		data.EndTime,
 		false,
 		getAllQuestionsForElectionChannel(data.Questions),
+		getAllAttendees(c.attendees),
 	}
 
 	// Add the SetupElection message to the new election channel
 	messageID := base64.URLEncoding.EncodeToString(msg.MessageID)
-	electionCh.inboxMu.Lock()
-	electionCh.inbox[messageID] = msg
-	electionCh.inboxMu.Unlock()
+	c.inboxMu.Lock()
+	c.inbox[messageID] = msg
+	c.inboxMu.Unlock()
```

Comment:
Maybe those change can be removed since it is already in #376.

Reply:
done